eng/pipelines/templates/stages/vmr-build.yml

```
@@ -35,6 +35,12 @@ parameters:
     name: NetCore-Public-XL
     demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
 
+variables:
+  isPR: $[eq(variables['Build.Reason'], 'PullRequest')]
+  isRelease: $[startsWith(variables['Build.SourceBranch'], 'refs/heads/release/')]
```

I think adding the `Branch` suffix would make this more readable where used.

```suggestion
  isReleaseBranch: $[startsWith(variables['Build.SourceBranch'], 'refs/heads/release/')]
```

---

Addressed via the changes explained in [this comment](https://github.com/dotnet/installer/pull/17370#discussion_r1332274551)